_posts/2022-11-20-jump_dev_2023.md

```diff
@@ -19,32 +18,35 @@ practitioners with interests in the methodological, algorithmic, and software as
 [JuMP](https://github.com/jump-dev/JuMP.jl) and related packages. In particular,
 we invite new contributors and those who have not met the core development team.
 
-## How do I attend?
+## Schedule
 
-To attend JuMP-dev, you must purchase a ticket to JuliaCon 2023. Ticket sales
-will open in early 2023.
+A preliminary schedule is as follows.
 
-All participants will uphold the [JuliaCon Code of Conduct](https://juliacon.org/2023/coc/).
+#### July 27
 
-## Travel support
+* All day: talks
+* Evening: JuliaCon conference dinner
 
-To encourage people to attend JuMP-dev, we have funds to provide travel support
-to a limited number of applicants.
+#### July 28
 
-We particularly encourage people who have not been to a JuMP-dev before to
-apply.
+* All day: talks
 
-[Apply for travel support](https://forms.gle/1pWBhu3VwBUophFV7)
+#### July 29
 
-The deadline for applications is January 31st.
+ * 0900-1400: hackathon
```

Comment:
```suggestion
 * 9h-14h: hackathon
```

Reply:
Ha. Adding an `h` after makes it even more confusing to NZ speakers. `09:00 - 14:00` or `9am - 2pm`?

Reply:
9am - 2pm is the least pretentious 😄

Reply:
Is 24-hr time pretentious???

Reply:
haha it's fine. people in america don't use the `h` Benoit suggested though

Reply:
9am - 2pm is fine then :laughing: